onnxruntime/core/providers/cuda/test/beam_search_topk.cc

```diff
@@ -1,11 +1,11 @@
 #ifndef NDEBUG
 
-#include "contrib_ops/cuda/transformers/beam_search_topk.h"
 #include <random>
 #include <numeric>
 #include <queue>
 #include <algorithm>
 #include <cuda_runtime.h>
+#include "contrib_ops/cuda/transformers/beam_search_topk.h"
```

Comment:
I think you should copy "#include <cuda_runtime.h>" to beam_search_topk.h instead.

Reply:
See: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

beam_search_topk.h should be on the top.